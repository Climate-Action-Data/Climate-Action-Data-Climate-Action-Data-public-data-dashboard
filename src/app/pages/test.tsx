'use client'
import { type NextPage } from "next";

const Test: NextPage = () => {
    return (
        <div>
            <h1>Hello</h1>
            <p>data</p>
        </div>
    )
}

export default Test