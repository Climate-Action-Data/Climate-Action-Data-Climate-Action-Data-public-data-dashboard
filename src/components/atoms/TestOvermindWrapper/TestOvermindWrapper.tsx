import { createOvermind } from 'overmind'
import { Provider } from 'overmind-react'
import { config } from '@/overmind'
import { PropsWithChildren } from 'react'

const overmind = createOvermind(config)

export const TestOvermindWrapper = (props: PropsWithChildren) => {
  return <Provider value={overmind}>{props.children}</Provider>
}
