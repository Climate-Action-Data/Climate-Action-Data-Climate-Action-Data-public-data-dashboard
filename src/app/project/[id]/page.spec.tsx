import { render } from '@testing-library/react'
import PageDetails from './page'
import { PROJECT_DETAIL } from '@/test/TestOvermindMockData'
import { TestOvermindWrapper } from '@/test/TestOvermindWrapper'

describe(`PageDetails`, () => {
  it(`renders correctly`, () => {
    const { container } = render(
      <TestOvermindWrapper>
        <PageDetails params={{ id: PROJECT_DETAIL.warehouseProjectId }} />
      </TestOvermindWrapper>,
    )
    expect(container).toMatchSnapshot()
  })
})
