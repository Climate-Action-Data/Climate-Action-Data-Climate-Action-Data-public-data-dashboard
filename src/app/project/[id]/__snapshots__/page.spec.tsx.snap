// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageDetails renders correctly 1`] = `
<div>
  <div
    class="css-14fdyiq"
  >
    <div
      class="css-0"
    >
      <div
        class="chakra-stack css-qlxuh7"
      >
        <svg
          class="chakra-icon css-g7llvd"
          focusable="false"
          viewBox="0 0 40 16"
        >
          <circle
            cx="1.42857"
            cy="1.42857"
            fill="currentFill"
            r="1.42857"
          />
          <circle
            cx="1.42857"
            cy="13.8097"
            fill="currentFill"
            r="1.42857"
          />
          <circle
            cx="13.8094"
            cy="1.42857"
            fill="currentFill"
            r="1.42857"
          />
          <circle
            cx="13.8094"
            cy="13.8097"
            fill="currentFill"
            r="1.42857"
          />
          <circle
            cx="26.1903"
            cy="1.42857"
            fill="currentFill"
            r="1.42857"
          />
          <circle
            cx="26.1903"
            cy="13.8097"
            fill="currentFill"
            r="1.42857"
          />
          <circle
            cx="38.5716"
            cy="1.42857"
            fill="currentFill"
            r="1.42857"
          />
          <circle
            cx="38.5716"
            cy="13.8097"
            fill="currentFill"
            r="1.42857"
          />
        </svg>
        <h2
          class="chakra-heading css-16o2lp8"
        >
          Project Details
        </h2>
      </div>
      <div
        class="css-1mliti3"
      >
        <div
          class="chakra-container css-tqikoc"
        >
          <div
            class="chakra-stack css-j7qwjs"
          >
            <div
              class="css-s8hfc8"
            >
              <div
                class="css-0"
              >
                <p
                  class="chakra-text css-1xi4xed"
                >
                  Standard
                </p>
                <div
                  class="css-0"
                >
                  <div
                    class="chakra-skeleton css-1tjwkx3"
                  />
                </div>
              </div>
              <div
                class="css-0"
              >
                <p
                  class="chakra-text css-1xi4xed"
                >
                  Methodology
                </p>
                <div
                  class="css-0"
                >
                  <div
                    class="chakra-skeleton css-1tjwkx3"
                  />
                </div>
              </div>
              <div
                class="css-0"
              >
                <p
                  class="chakra-text css-1xi4xed"
                >
                  Project Developer
                </p>
                <div
                  class="css-0"
                >
                  <div
                    class="chakra-skeleton css-1tjwkx3"
                  />
                </div>
              </div>
              <div
                class="css-0"
              >
                <p
                  class="chakra-text css-1xi4xed"
                >
                  Project Link
                </p>
                <div
                  class="css-0"
                >
                  <div
                    class="chakra-skeleton css-1tjwkx3"
                  />
                </div>
              </div>
              <div
                class="css-0"
              >
                <p
                  class="chakra-text css-1xi4xed"
                >
                  Sector
                </p>
                <div
                  class="css-0"
                >
                  <div
                    class="chakra-skeleton css-1tjwkx3"
                  />
                </div>
              </div>
              <div
                class="css-0"
              >
                <p
                  class="chakra-text css-1xi4xed"
                >
                  Project Type
                </p>
                <div
                  class="css-0"
                >
                  <div
                    class="chakra-skeleton css-1tjwkx3"
                  />
                </div>
              </div>
              <div
                class="css-0"
              >
                <p
                  class="chakra-text css-1xi4xed"
                >
                  Status
                </p>
                <div
                  class="css-0"
                >
                  <div
                    class="chakra-skeleton css-1tjwkx3"
                  />
                </div>
              </div>
              <div
                class="css-0"
              >
                <p
                  class="chakra-text css-1xi4xed"
                >
                  Status Updated On
                </p>
                <div
                  class="css-0"
                >
                  <div
                    class="chakra-skeleton css-1tjwkx3"
                  />
                </div>
              </div>
            </div>
            <div
              class="chakra-stack__divider css-1g2u1yc"
            />
            <div
              class="css-k008qs"
            >
              <div
                class="css-s8hfc8"
              >
                <div
                  class="css-0"
                >
                  <p
                    class="chakra-text css-1xi4xed"
                  >
                    Total Available Units
                  </p>
                  <div
                    class="css-0"
                  >
                    <div
                      class="chakra-skeleton css-1tjwkx3"
                    />
                  </div>
                </div>
                <div
                  class="css-0"
                >
                  <p
                    class="chakra-text css-1xi4xed"
                  >
                    Total Issuances
                  </p>
                  <div
                    class="css-0"
                  >
                    <div
                      class="chakra-skeleton css-1tjwkx3"
                    />
                  </div>
                </div>
                <div
                  class="css-0"
                >
                  <p
                    class="chakra-text css-1xi4xed"
                  >
                    Crediting Period
                  </p>
                  <div
                    class="css-0"
                  >
                    <div
                      class="chakra-skeleton css-1tjwkx3"
                    />
                  </div>
                </div>
              </div>
              <div
                class="css-s8hfc8"
              >
                <div
                  class="css-0"
                >
                  <p
                    class="chakra-text css-1xi4xed"
                  >
                    Total Retirements
                  </p>
                  <div
                    class="css-0"
                  >
                    <div
                      class="chakra-skeleton css-1tjwkx3"
                    />
                  </div>
                </div>
                <div
                  class="css-0"
                >
                  <p
                    class="chakra-text css-1xi4xed"
                  >
                    Annual Estimated Units
                  </p>
                  <div
                    class="css-0"
                  >
                    <div
                      class="chakra-skeleton css-1tjwkx3"
                    />
                  </div>
                </div>
                <div
                  class="css-0"
                >
                  <p
                    class="chakra-text css-1xi4xed"
                  >
                    Unit Metric Measurement
                  </p>
                  <div
                    class="css-0"
                  >
                    <div
                      class="chakra-skeleton css-1tjwkx3"
                    />
                  </div>
                </div>
              </div>
            </div>
            <div
              class="chakra-stack__divider css-1g2u1yc"
            />
            <div
              class="css-s8hfc8"
            >
              <div
                class="css-0"
              >
                <p
                  class="chakra-text css-1xi4xed"
                >
                  NDC Coverage
                </p>
                <div
                  class="css-0"
                >
                  <div
                    class="chakra-skeleton css-1tjwkx3"
                  />
                </div>
              </div>
              <div
                class="css-0"
              >
                <p
                  class="chakra-text css-1xi4xed"
                >
                  Correspondingly Adjusted
                </p>
                <div
                  class="css-0"
                >
                  <div
                    class="chakra-skeleton css-1tjwkx3"
                  />
                </div>
              </div>
            </div>
            <div
              class="chakra-stack__divider css-1g2u1yc"
            />
            <div
              class="css-s8hfc8"
            >
              <div
                class="css-0"
              >
                <p
                  class="chakra-text css-1xi4xed"
                >
                  Project Tags
                </p>
                <div
                  class="css-0"
                >
                  <div
                    class="chakra-skeleton css-1tjwkx3"
                  />
                </div>
              </div>
              <div
                class="css-0"
              >
                <p
                  class="chakra-text css-1xi4xed"
                >
                  Co-Benefits
                </p>
                <div
                  class="css-0"
                >
                  <div
                    class="chakra-skeleton css-1tjwkx3"
                  />
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="chakra-container css-1b7mn2k"
        >
          <div
            class="chakra-skeleton css-t8wcef"
          />
          <div
            class="chakra-container css-bybdqv"
          >
            <div
              class="css-s8hfc8"
            >
              <div
                class="css-0"
              >
                <p
                  class="chakra-text css-1xi4xed"
                >
                  Country
                </p>
                <p
                  class="chakra-text css-0"
                />
              </div>
              <div
                class="css-0"
              >
                <p
                  class="chakra-text css-1xi4xed"
                >
                  In-country Region
                </p>
                <p
                  class="chakra-text css-0"
                />
              </div>
              <div
                class="css-0"
              >
                <p
                  class="chakra-text css-1xi4xed"
                >
                  Geographic identifier
                </p>
                <p
                  class="chakra-text css-0"
                >
                  --
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="css-0"
    >
      <div
        class="chakra-stack css-qlxuh7"
      >
        <svg
          class="chakra-icon css-g7llvd"
          focusable="false"
          viewBox="0 0 40 16"
        >
          <circle
            cx="1.42857"
            cy="1.42857"
            fill="currentFill"
            r="1.42857"
          />
          <circle
            cx="1.42857"
            cy="13.8097"
            fill="currentFill"
            r="1.42857"
          />
          <circle
            cx="13.8094"
            cy="1.42857"
            fill="currentFill"
            r="1.42857"
          />
          <circle
            cx="13.8094"
            cy="13.8097"
            fill="currentFill"
            r="1.42857"
          />
          <circle
            cx="26.1903"
            cy="1.42857"
            fill="currentFill"
            r="1.42857"
          />
          <circle
            cx="26.1903"
            cy="13.8097"
            fill="currentFill"
            r="1.42857"
          />
          <circle
            cx="38.5716"
            cy="1.42857"
            fill="currentFill"
            r="1.42857"
          />
          <circle
            cx="38.5716"
            cy="13.8097"
            fill="currentFill"
            r="1.42857"
          />
        </svg>
        <h2
          class="chakra-heading css-16o2lp8"
        >
          Verification & Validation
        </h2>
      </div>
      <div
        class="css-1mliti3"
      >
        <div
          class="chakra-container css-kzfr2u"
        >
          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque vestibulum ex vel blandit congue. Integer turpis lectus, porttitor sed tellus et, rhoncus suscipit lorem. Donec quis ante vitae mi commodo ultrices. Pellentesque accumsan magna enim, nec venenatis sapien vestibulum vitae.
        </div>
      </div>
    </div>
    <div
      class="css-0"
    >
      <div
        class="chakra-stack css-qlxuh7"
      >
        <svg
          class="chakra-icon css-g7llvd"
          focusable="false"
          viewBox="0 0 40 16"
        >
          <circle
            cx="1.42857"
            cy="1.42857"
            fill="currentFill"
            r="1.42857"
          />
          <circle
            cx="1.42857"
            cy="13.8097"
            fill="currentFill"
            r="1.42857"
          />
          <circle
            cx="13.8094"
            cy="1.42857"
            fill="currentFill"
            r="1.42857"
          />
          <circle
            cx="13.8094"
            cy="13.8097"
            fill="currentFill"
            r="1.42857"
          />
          <circle
            cx="26.1903"
            cy="1.42857"
            fill="currentFill"
            r="1.42857"
          />
          <circle
            cx="26.1903"
            cy="13.8097"
            fill="currentFill"
            r="1.42857"
          />
          <circle
            cx="38.5716"
            cy="1.42857"
            fill="currentFill"
            r="1.42857"
          />
          <circle
            cx="38.5716"
            cy="13.8097"
            fill="currentFill"
            r="1.42857"
          />
        </svg>
        <h2
          class="chakra-heading css-16o2lp8"
        >
          Issuances & Retirements
        </h2>
      </div>
      <div
        class="css-1mliti3"
      >
        <div
          class="chakra-container css-kzfr2u"
        >
          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque vestibulum ex vel blandit congue. Integer turpis lectus, porttitor sed tellus et, rhoncus suscipit lorem. Donec quis ante vitae mi commodo ultrices. Pellentesque accumsan magna enim, nec venenatis sapien vestibulum vitae.
        </div>
      </div>
    </div>
    <div
      class="css-0"
    >
      <div
        class="chakra-stack css-qlxuh7"
      >
        <svg
          class="chakra-icon css-g7llvd"
          focusable="false"
          viewBox="0 0 40 16"
        >
          <circle
            cx="1.42857"
            cy="1.42857"
            fill="currentFill"
            r="1.42857"
          />
          <circle
            cx="1.42857"
            cy="13.8097"
            fill="currentFill"
            r="1.42857"
          />
          <circle
            cx="13.8094"
            cy="1.42857"
            fill="currentFill"
            r="1.42857"
          />
          <circle
            cx="13.8094"
            cy="13.8097"
            fill="currentFill"
            r="1.42857"
          />
          <circle
            cx="26.1903"
            cy="1.42857"
            fill="currentFill"
            r="1.42857"
          />
          <circle
            cx="26.1903"
            cy="13.8097"
            fill="currentFill"
            r="1.42857"
          />
          <circle
            cx="38.5716"
            cy="1.42857"
            fill="currentFill"
            r="1.42857"
          />
          <circle
            cx="38.5716"
            cy="13.8097"
            fill="currentFill"
            r="1.42857"
          />
        </svg>
        <h2
          class="chakra-heading css-16o2lp8"
        >
          Documents
        </h2>
      </div>
      <div
        class="css-1mliti3"
      >
        <div
          class="chakra-container css-kzfr2u"
        >
          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque vestibulum ex vel blandit congue. Integer turpis lectus, porttitor sed tellus et, rhoncus suscipit lorem. Donec quis ante vitae mi commodo ultrices. Pellentesque accumsan magna enim, nec venenatis sapien vestibulum vitae.
        </div>
      </div>
    </div>
  </div>
</div>
`;
